### PR TITLE
[Console] Adds php version to console output

### LINKED ISSUE #34656: [Console] display PHP version and path to binary
**Description**  
PHP version used to run Symfony Console may be different from the one used in the webserver for whatever reason (e.g. improper configuration when maintaining several available versions locally).

It would be useful to see which PHP version and from where is used. I know you can check the version using the `about` command, but it would be more convenient to see it each time.

What do you think? Should I prepare a PR?

**Example**  
image

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes 
| Deprecations? | no 
| Tickets       | Fix #34656 
| License       | MIT
| Doc PR        | tbd. 

This PR will add the current used PHP Version to the console output when running `php bin/console` and `php bin/console --version`.

### CODE REVIEW

**Comment by stof** [comment]
Do we really want to display this info for the `--version` display of **all** tools built on top of symfony/console ? I'm not convinced about it. Thus, the PHP version is not something related to the tool. There is already `php --version` displaying it.

**Comment by noniagriconomie** [comment]
it is already present in the `bin/console about`
i think it will be to much to add such info :/

File: src/Symfony/Component/Console/Application.php [hunk 1]
```diff
@@ -1184,4 +1184,9 @@ private function init()
             $this->add($command);
         }
     }
+
+    private function getPhpVersion(): string
+    {
+        return sprintf('PHP <info>%s</info> (binary: %s)', PHP_VERSION, PHP_BINARY);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Application.php]: I wouldn't add the path to the binary as that can be quite long.
For the version, this constant contains the extra part, which is noise mostly.
Let's use `PHP_*_VERSION` instead.

**Review by nicolas-grekas — APPROVED** [review]
I think that can help and doesn't hurt. Good to me.

**Comment by fabpot** [comment]
I'm :-1: as well. I don't like duplicating information that is already simple to get in another way.

**Pull request closed without merging**